python/tvm/topi/cuda/dense.py

```diff
@@ -42,11 +42,8 @@ def dense_cublas(cfg, data, weight, bias=None, out_dtype=None):
     batch, in_dim = data.shape
     out_dim, _ = weight.shape
     matmul = cublas.matmul(data, weight, False, True)
-    if isinstance(batch, int):
+    if isinstance(batch, int) and isinstance(in_dim, int) and isinstance(out_dim, int):
```

Comment:
What if batch has `tir.IntImm` type?

Reply:
Thanks. I'm confusing about this, under what circumstances will batch be tir.IntImm ?

Reply:
It usually happens when the shape comes from C++ via FFI, so it'd better to make sure we cover both cases.

Reply:
@comaniac Thanks, I'll fix it later

Reply:
@comaniac I haven't found a way yet to do assertion inside te's `compute` or `schedule`, and left this change to the future work, thanks!